code/client/src/utils/authenticateUser.js

```diff
@@ -0,0 +1,27 @@
+// utils/authenticateUser.js
+export async function authenticateUser(credentials, userDatabase) {
```

Comment:
the testing should be done on the existing authenticate function, not a newly created one